RuslanImankulov/homework1.js

```diff
@@ -0,0 +1,61 @@
+//Func to check palindrome
+const isPalindrome = (str) => {
+    let i = 0;
+    let j = str.length -1;
+    for (let z=1; z <= str.length/2; ++z) {
+        if (str[i] !== str[j]) {
+            return false
+        };
+        ++i;
+        --j;
+    }
+    return true
+  }
+  
+//Func to check palindrome v1
+  const isPalindrome_v1 = (str) => {
+    if (str.length === 1) {
+      return true
+    } else{
```

Comment:
It's unnecessary nesting, because when length will be 1 no other code will be executed. So move the code from `else` block and put it just after the first `if`.